### PR TITLE
Fix blog tags polluting the website

### PR DESCRIPTION
**Pull request by Fryguy**
The tags listed in #969 included duplicates, commas, and special
characters, which was messing up the website.  This corrects it.

I've kept the cloudforms tag, but removed the redundant cf, cfme, cf4,
cf4 tags.  I've also removed the confusing uncategorized tag.

@jrafanie Please review.

Screenshot of the problem:
ManageIQ_-_ManageIQ_Blog_and___dev_manageiq_org

### CODE REVIEW

File: site/_posts/2014-07-08-integration-with-travis.html.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Integration with Travis CI
 date: 2014-07-08 14:55 UTC
 author: chessbyte
-tags: collaboration announcements
+tags: collaboration announcements
```
Comment (Fryguy) [inline site/_posts/2014-07-08-integration-with-travis.html.md:5]: This one can't be seen in the diff, but there was a trailing `\r\n` here.

File: site/_posts/2013-11-20-inspectxml-dump-objects-as-xml.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: inspectXML – Dump objects as XML
 date: 2013-11-20
-tags: external operations cloudforms, vmware, rhev, kvm, cluster, vm, datastore, cf2, cf3, cfme, XML, XML format
```
Comment (Fryguy) [inline site/_posts/2013-11-20-inspectxml-dump-objects-as-xml.md:4]: This one can't been seen in the diff, but there are embedded special characters in this line:
__dev_manageiq_org_site__posts_2013-11-20-inspectxml-dump-objects-as-xml_md_—_manageiq_org

**Comment by Fryguy** [comment]
I'm not sure how they are generated, but I would presume so.  If I had to guess they are ordered in the order they are found in the blog posts themselves.

**Comment by chessbyte** [comment]
something like https://mindthecode.com/blog/sorting-tags-in-jekyll applied to https://github.com/ManageIQ/manageiq.org/blob/master/site/_plugins/blog_tags.rb will probably do the trick

**Pull request merged**